src/app/services/sms/sms.types.ts

```diff
@@ -88,6 +91,7 @@ export interface IBouncedSubmissionSmsCountSchema
 
 export interface ISmsCountModel extends Model<ISmsCountSchema> {
   logSms: (logParams: LogSmsParams) => Promise<ISmsCountSchema>
+  retrieveFreeSmsCounts: (userId: string) => Promise<number>
```

Comment:
can we add JSDocs for this? (realized logSms has missing JSDocs, but yeah just add for this one to prevent PR from going out of scope)

Reply:
sure, added!